pkgs/test/lib/src/runner/browser/post_message_channel.dart

```diff
@@ -16,8 +16,10 @@ StreamChannel<Object?> postMessageChannel() {
   dom.window.console.log('Suite starting, sending channel to host'.toJS);
   var controller = StreamChannelController<Object?>(sync: true);
   var channel = dom.createMessageChannel();
-  dom.window.parent
-      .postMessage('port', dom.window.location.origin, [channel.port2]);
+  dom.window.parent.postMessage(
+      {'messageType': 'port', 'href': dom.window.location.href},
+      dom.window.location.origin,
+      [channel.port2]);
```

Comment:
This change is not backwards compatible. A compatible `host.dart` needs to be rolled out everywhere before we can land this change.

Reply:
Hmm, that won't work because we're transferring the port.